package.json

```diff
@@ -51,6 +51,7 @@
   "devDependencies": {
     "@playwright/test": "^1.32.2",
     "@swc/core": "^1.3.49",
+    "@types/jest": "^29.5.4",
```

Comment:
This package is needed so that I can type-check a test file. (It adds jest functions like `describe` and `it` to the global context, so that TypeScript doesn't complain about them.)